spec/models/user_cleaner_spec.rb

```diff
@@ -1,19 +1,187 @@
 require "rails_helper"
 
 RSpec.describe(UserCleaner, type: :model) do
-  it "has a factory" do
-    expect(FactoryBot.build(:user_cleaner))
-      .to be_kind_of(UserCleaner)
+  describe("#set/unset_deletion_date") do
```

Comment:
Technically, `UserCleaner` has now become a worker instead of a model because `user_cleaner.rb` is placed in the workers folder. Would it not be more logical to place the `user_cleaner.rb` file in the models folder and rename the worker to `user_cleaner.rb` (the worker folder contains jobs by design)?

Reply:
When I started with this PR, the UserCleaner was indeed in the `models` folder. I changed its location since I didn't see why we would consider such a class as a model in the usual ORM kind of view where database fields are modeled in object-oriented code.

That's why I've placed it in the `workers` folder as that seemed to be the best place for it as far as I could see. Maybe you know a better place that would fit better? But what's wrong with having the actual job implementation next to file that defines the job and calls the implementation?

Reply:
(Note that you can also attach PR comments to files themselves instead of specific code lines.)

Reply:
As we discussed last week, it should be moved to the `models` for uniformity with the other workers. Changing the overall placing of PORO files would be a different PR for later.

Reply:
Done in d65228ae3 and f1184867d